src/components/Video.tsx

```diff
@@ -1,27 +1,39 @@
 import { FunctionComponent } from 'react'
 
-export const Video: FunctionComponent<{ filePath: string }> = ({ filePath }) => (
+interface Video {
+    source: {
+        webm: string
+        mp4: string
+    }
+    loop: boolean
+    caption: string // can also be used for alt
```

Comment:
I must have been thinking of images when I suggested this haha. But we can definitely use this for image assets that have captions as a future improvement. We can also use `figure` for them as well.

```suggestion
    caption: string
```